libs/designsystem/src/lib/components/dropdown/dropdown.component.scss

```diff
@@ -64,57 +63,73 @@ $margin-horizontal-total: 2 * size('s');
   }
 }
 
+kirby-popover {
+  --max-height: #{$dropdown-max-height};
+}
+
 kirby-card {
-  display: none;
-  opacity: 0;
-  position: absolute;
-  width: 100%;
   max-height: $dropdown-max-height;
   margin-top: size('xxxs');
+  margin-bottom: size('xxxs');
   overflow-y: auto;
   box-shadow: get-elevation(8);
-  z-index: z('dropdown');
+  min-width: 320px - $margin-horizontal-total;
+  @include media('>=small') {
+    min-width: 375px - $margin-horizontal-total;
+  }
+  max-width: calc(100vw - #{$margin-horizontal-total});
 }
 
-:host(.is-opening) {
+:host(:not(.with-popover)) {
```

Comment:
I suggest inverting the logic to clarify. The cognitive load is higher when processing "not".